lib/network/modules/NodesHandler.js

```diff
@@ -285,7 +287,9 @@ class NodesHandler {
       let data = changedData[i];
       if (node !== undefined) {
         // update node
-        dataChanged = node.setOptions(data);
+        if (node.setOptions(data) === true) {  // explicit test because setOptions() can return undefined
```

Comment:
`Node.setOptions()` can return `undefined`.

...yes, very ugly. Let's see if we can change that.